demo/paddle/TUTORIAL_EN.md

```diff
@@ -0,0 +1,171 @@
+# How to use VisualDL in PaddlePaddle
+
+Here we will show you how to use VisualDL with PaddlePaddle so that you can visualize the training process of PaddlePaddle.
+We will use the Paddle Convolution Neural Network to train the [Cifar10](https://www.cs.toronto.edu/~kriz/cifar.html) dataset as an example.
+
+This example is the modification with fluid PaddlePaddle's API from this official Paddle Book
+[Example](https://github.com/PaddlePaddle/book/tree/develop/03.image_classification)
+
+The full demonstration code can be downloaded in [here](https://github.com/PaddlePaddle/VisualDL/blob/develop/demo/paddle/paddle_cifar10.py).
+
+The script is based on Paddle v2 0.11. You can do ```pip install paddlepaddle``` or ```docker pull paddlepaddle/paddle:0.11.0```. Notice Paddle does not support Python3 yet. For details, please
```

Comment:
Another pitfall is that, Paddle is using protobuf 3.1 and VisualDL is using protobuf 3.5. Should should make sure that they are using the latest protobuf. Otherwise, they will get error whiling import the visualdl.

Reply:
ya, I got the same issue, will update that